docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -0,0 +1,91 @@
+================
+RPM package
+================
+
+Users can install Trino using the RPM Package Manager (RPM) on Linux
+distributions that support RPM.
+
+The RPM archive contains the application, all plugins, the necessary default
+configuration files, default setups, and integration with the operating system
+to start as a service.
+
+Installing Trino
+----------------
+
+Download the Trino server RPM package :maven_download:`server-rpm`. Use the
+``rpm`` command to install the package:
+
+.. code-block:: text
+
+    rpm -i trino-server-rpm-*.rpm --nodeps
+
+Installing the :ref:`required Java and Python setup <requirements>` is typically
+managed separately.
+
+Service script
+--------------
+
+The RPM installation deploys a service script configured with ``chkconfig`` so
+that the service can be started automatically on operating system boot. After
+installation, you can manage the Trino server with the ``service`` command:
+
+.. code-block:: text
+
+    service trino [start|stop|restart|status]
+
+* ``start`` Starts the server as a daemon and returns the process ID.
+* ``stop`` Shuts down the server started with either ``start`` or ``run``. Sends
+  the SIGTERM signal.
+* ``restart`` Stops and then starts a running server, or starts a stopped
+  server, assigning a new process ID.
+* ``status`` Prints a status line, either *Stopped pid* or *Running as pid*.
+
+Installation directory structure
+--------------------------------
+
+The RPM package places the various files used by Trino in accordance with the
+Linux Filesystem Hierarchy Standard. This differs from the default tarball
+installation of Trino, where all folders are in the installation directory. For
+example, with a ``tar.gz`` file, configuration files are located by default in the
+``etc`` folder of the installation directory. By contrast, the RPM package
+installation uses ``/etc/trino`` for the same purpose.
+
+The RPM installation places Trino files using the following directory
+structure:
+
+* ``/usr/lib/trino/lib/``: Contains the various libraries needed to run the
+  product. Plugins go in a ``plugin`` subdirectory.
+* ``/etc/trino``: Contains the general Trino configuration files like
+  ``node.properties``, ``jvm.config``, ``config.properties``. Catalog
+  configurations go in a ``catalog`` subdirectory.
+* ``/etc/trino/env.sh``: Java installation path used by Trino.
+* ``/var/log/trino``: Contains the log files.
+* ``/var/lib/trino/data``: This is the data directory.
+* ``/usr/shared/doc/trino``: Contains documentation.
+* ``/etc/rc.d/init.d/trino``: Contains the service scripts for controlling the
+  server process.
+
+RPM-specific configuration settings
+-----------------------------------
+
+The node.properties file requires defining the following two additional
```

Comment:
This is not set by default?

Reply:
I dont think so .. need to check but I think the file might not be created by default?  Please verify @sheajamba

Reply:
There's this: https://github.com/trinodb/trino/blob/master/core/trino-server-rpm/src/main/resources/dist/config/node.properties

Reply:
And it gets modified by the post-install script to generate a unique node id:
```
[root@b0159cfd5e23 yum.repos.d]# cat /etc/trino/node.properties
node.environment=test
node.id=f360176d-5595-43c7-99f6-bb1e96403e27
node.data-dir=/var/lib/trino/data
catalog.config-dir=/etc/trino/catalog
node.server-log-file=/var/log/trino/server.log
node.launcher-log-file=/var/log/trino/launcher.log
```

Reply:
perfect ... so we can remove this ..